lib/widgets/shipanther_scaffold.dart

```diff
@@ -153,6 +187,17 @@ List<Widget> drawerItemsFor(BuildContext context, api.User user) {
       ),
     );
   }
+  // widgets.add(
```

Comment:
This is just noise. It should be removed from PRs as it leads to unnecessary cycle.s